awx_collection/README.md

```diff
@@ -77,14 +77,6 @@ The build target `make build_collection` will template out a `galaxy.yml` file
 with automatic detection of the current AWX version. Then it builds the
 collection with the `ansible-galaxy` CLI.
 
-## Roadmap
-
-Major future development items on the agenda include:
-
- - Removing tower-cli as a dependency
- - Renaming the modules, for example `tower_organization` to just `organization`
-   and giving a deprecation period for the switch
```

Comment:
This isn't really on the agenda now (we could revisit it, but I wouldn't say the decision is made)

and this isn't the best place, we should handle these in issues.